Resources/Audio/Effects/attributions.yml

```diff
@@ -216,3 +216,8 @@
   copyright: Bhijn and Myr (github username deathride58)
   license: CC0-1.0
   source: https://github.com/space-wizards/space-station-14/pull/23212
+
+- files: ["metal_pipe_falling.ogg"]
+  license: "CC0-1.0"
+  copyright: "jixaw"
+  source: "https://www.youtube.com/watch?v=iDLmYZ5HqgM"
```

Comment:
It's a stock sound from prior to 1993 with unknown provenance. I wanted to add it in as a round restart sfx :(